data/Dataset.py

```diff
@@ -0,0 +1,21 @@
+import torch
+import PIL
+class LandUseDataset(torch.utils.data.Dataset):
```

Comment:
Since I am going to change how the processed data info is stored (currently in .csv, will change to json, with each image having its label a one-hot-vector already, so there is no need for further processing)